ADOPTERS.md

```diff
@@ -5,4 +5,5 @@ This is a likely incomplete list of KubeVirt adopters - end-users and distributo
 | User | Since | Use-Case |
 |-|-|:-|
 | Red Hat | 2016 | As a distributor we are building OpenShift Virtualization on KubeVirt in order to enable VM workloads and -flows on Kubernetes. |
-| [Kubermatic](https://kubermatic.com) | 2019 | As a distributor we are running KubeVirt to enable VM workload on Kubermatic Virtualization. Follow the [link](https://www.kubermatic.com/products/kubevirt/) for more information. |
\ No newline at end of file
+| [Kubermatic](https://kubermatic.com) | 2019 | As a distributor we are running KubeVirt to enable VM workload on Kubermatic Virtualization. Follow the [link](https://www.kubermatic.com/products/kubevirt/) for more information. |
+| H3C | 2019 | As a distributor we are building KubeVirt in order to enable VM workloads on Kubernetes. |
```

Comment:
@zxb789 would you care to elaborate about your use case a bit more?

Red Hat and Kubermatic are distributing KubeVirt to third parties. Is h3c a distributor as well? If so, can you share a URL for how can one download it?

BTW do you know why you show up twice on [cncf|https://github.com/cncf/landscape/blob/master/landscape.yml] ?

Reply:
our product name "cloudos " focus on private cloud ;and distributing to some company who need it,so it cann't download from website。

Reply:
sorry, the second question i don't know why has twice on landscape。it cloud be show up only one times.  thanks